### PR TITLE
Add Certification SIG

### PR DESCRIPTION
**Pull request by grooverdan**

### CODE REVIEW

File: docs/sigs/Certification.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* Review of the AlmaLinux OS Foundation Hardware Certification Program document to ensure its clear and concise.
+* Contributions towards expanding the AlmaLinux Hardware Certification Suite to represent a meaningful range of hardware and Linux hardware driver tests that an end user would expect a certification to cover.
+* There is work in progress happening where an Independent Software Vendor can maintain a set of tests for their software with obligations as an addition part of this suite. Once this is defined more, vendor participation will be encouraged. Let us know if this is you so you can be advised when we are ready.
+* Eventually users will be welcome to run the AlmaLinux Hardware Certification Suite on the hardware they have access to and submit these results.
+
+## SIG members
+
+* [Jonathan Wright](mailto:jonathan@almalinux.org) - Committee lead
+* [Akitsu Izawa](mailto:akitsu.izawa@cybertrust.co.jp) - member
+* [Asaka Matsumoto](mailto:asaka.matsumoto@cybertrust.co.jp) - member
```
Comment (metalefty) [inline docs/sigs/Certification.md:39]: Asaka is presently on leave of absence. I am the temporary successor of her. 
Could you add myself instead? 
- Koichiro Iwao <meta@almalinux.org>

Comment (grooverdan) [reply docs/sigs/Certification.md:39]: done!

Comment (metalefty) [reply docs/sigs/Certification.md:39]: Asaka Matsumoto is on leave.

**Comment by bennyvasquez** [comment]
Just stopping by to note: this SIG was already approved before our SIG creation process during our [September 2023 board meeting](https://wiki.almalinux.org/Transparency.html#minutes-of-almalinux-os-foundation-board-meetings). Once @AlmaLinux/certification-sig folks are okay with the draft here, tag me in and I'll get this PR reviewed/merged in to make it official.

File: docs/sigs/Certification.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Certification SIG"
+---
+# name of SIG
+
+Create and maintain standards and procedures related to hardware and software certification for AlmaLinux.
+
+## How to Join
+
+Email a SIG member or show up on the chat. Occasionally there non-disclosure agreements with hardware vendors, so some chats/meeting might be private, but SIG members will do their best to welcome everyone as fully as possible.
```
Comment (jonathanspw) [inline docs/sigs/Certification.md:10]:
```suggestion
Email a SIG member or show up on the chat. Occasionally there are non-disclosure agreements with hardware vendors, so some chats/meeting might be private, but SIG members will do their best to welcome everyone as fully as possible.
```

**Review by jonathanspw — CHANGES_REQUESTED** [review]

**Review by metalefty — CHANGES_REQUESTED** [review]

File: docs/sigs/Certification.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* Review of the AlmaLinux OS Foundation Hardware Certification Program document to ensure its clear and concise.
+* Contributions towards expanding the AlmaLinux Hardware Certification Suite to represent a meaningful range of hardware and Linux hardware driver tests that an end user would expect a certification to cover.
+* There is work in progress happening where an Independent Software Vendor can maintain a set of tests for their software with obligations as an addition part of this suite. Once this is defined more, vendor participation will be encouraged. Let us know if this is you so you can be advised when we are ready.
+* Eventually users will be welcome to run the AlmaLinux Hardware Certification Suite on the hardware they have access to and submit these results.
+
+## SIG members
+
+* [Jonathan Wright](mailto:jonathan@almalinux.org) - Committee lead
+* [Koichiro Iwao](mailto:meta@almalinux.org) - member
+* [Asaka Matsumoto](mailto:asaka.matsumoto@cybertrust.co.jp) - member
```
[the remark this answers is not in the document]
Comment (grooverdan) [reply docs/sigs/Certification.md]: Appoligies, I should have being paying more attention on late night edits.

**Comment by sej7278** [comment]
:+1: looks good to me

**Review by jonathanspw — APPROVED** [review]
lgtm

**Comment by jonathanspw** [comment]
@bennyvasquez ready to go.

**Comment by bennyvasquez** [comment]
AH! Just noticed that this PR doesn't add it to the menu list. Mind adding that @jonathanspw or @grooverdan? Then we'll be good to go.

**Comment by grooverdan** [comment]
Menu list item added

**Review by bennyvasquez — APPROVED** [review]

**Pull request merged**